kitsune/search/v2/es7_utils.py

```diff
@@ -49,3 +53,29 @@ def es_analyzer_for_locale(locale):
 def es7_client():
     """Return an ES7 Elasticsearch client"""
     return elasticsearch7.Elasticsearch(settings.ES7_URLS)
+
+
+def get_doc_types(paths=['kitsune.search.v2.documents']):
+    """Return all registered document types"""
+
+    doc_types = []
+    modules = [importlib.import_module(path) for path in paths]
+
+    for module in modules:
+        for key in dir(module):
+            cls = getattr(module, key)
+            if inspect.isclass(cls) and issubclass(cls, Document) and cls != Document:
```

Comment:
Do we need to add other types here like `InnerDocument`?

Reply:
I don't know. This was a best effort approach to discover stuff. It can be improved in the future.